tests/aws/services/cloudformation/api/test_stacks.py

```diff
@@ -642,3 +651,94 @@ def test_blocked_stack_deletion(aws_client, cleanups, snapshot):
     snapshot.match("stack_post_success_delete", stack_post_success_delete)
     stack_events = cfn.describe_stack_events(StackName=stack_id)
     snapshot.match("stack_events", stack_events)
+
+
+MINIMAL_TEMPLATE = """
+Resources:
+    SimpleParam:
+        Type: AWS::SSM::Parameter
+        Properties:
+            Value: test
+            Type: String
+"""
+
+MINIMAL_TEMPLATE_UPDATED = """
+Resources:
+    SimpleParam:
+        Type: AWS::SSM::Parameter
+        Properties:
+            Value: test2
+            Type: String
+"""
+
+
+@markers.snapshot.skip_snapshot_verify(
+    paths=["$..EnableTerminationProtection", "$..LastUpdatedTime"]
+)
+@markers.aws.validated
+def test_name_conflicts(aws_client, snapshot, cleanups):
```

Comment:
Nice, a good clear test outlining the functionality. Good to explore what the AWS functionality is here as well!